### PR TITLE
[mobile] Add a wizard to configure app

### PR DESCRIPTION
**Pull request by enguerran**
I could split this onto 4 commits, but it's kinda one thing. So I push it as it.

Featuring:
- [X] wrap the main application component (react-router)
- [X] add a new reducer (redux)
- [X] add some style (stylus)
- [X] add i18n (transifex)
- [x] add tests
- [x] check the linter (aka pass the tests)

### CODE REVIEW

File: mobile/src/containers/OnBoarding.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import { translate } from '../../../src/lib/I18n'
+
+import styles from '../styles/onboarding'
+
+const OnBoarding = ({t, isLoggedIn, onClick, children}) => {
+  if (!isLoggedIn) {
+    return (
+      <div class={classnames(styles['wizard'])}>
+        <p>{t('wizard.cozy_address')}</p>
+        <input type="text" placeholder={t('wizard.cozy_address_placeholder')} />
```
Comment (kosssi) [inline mobile/src/containers/OnBoarding.jsx]: replace " to '

File: mobile/src/containers/OnBoarding.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <p>{t('wizard.description')}</p>
+        <button role="button" className="coz-btn coz-btn--regular" onClick={onClick}>{t('wizard.next')}</button>
```
Comment (kosssi) [inline mobile/src/containers/OnBoarding.jsx]: replace " to '

File: mobile/src/reducers/mobile.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const initialState = {
+  loggedIn: false
+}
+
+export const mobile = (state = initialState, action) => {
+  switch(action.type) {
```
Comment (kosssi) [inline mobile/src/reducers/mobile.js]: replace 'switch(' to 'switch ('

**Comment by nono** [comment]
The build on travis has failed because of the JS linter (standard). Can you look at it? It's probably a very minor thing.

And, do you plan to add tests for this feature?

**Comment by enguerran** [comment]
Yes, I may have tests and linting before submitting.
I'll correct that and be back soon.

**Comment by enguerran** [comment]
Please, could you validate below changes:
- include mobile translations in the root `src/locales/en.json` or may we include then in `mobile/` folder?
- rewrite a mobile reducers to combine all needed reducers?

**Comment by enguerran** [comment]
@kosssi could you review your review?

**Comment by enguerran** [comment]
@goldoraf and others: could we discuss about this two options (I choosen the first one in this PR):

```
import AppRoute from '../../src/components/AppRoute'

const OnBoarding = ({ isLoggedIn, children }) => {
  if (!isLoggedIn) {
    return (<p>You must log in first</p>)
  }
  return children
}

<Route component={OnBoarding}>
  {AppRoute}
</Route>
```

or

```
import auth from './auth'

const OnBoarding = () => (<p>You must log in first</p>)

function checkLoggedIn() {
  if (!auth.loggedIn()) {
    replace({
      pathname: '/onboarding',
      state: { nextPathname: nextState.location.pathname }
    })
  }
}

<Route component={App} onEnter={checkLoggedIn}>
  <Redirect from='/' to='files' />
  <Route path='files(/:file)' component={Folder} />
</Route>
<Route path='onboarding' component='OnBoarding' />
```

_or something approching…_

**Comment by enguerran** [comment]
Do you want me to squash those multiple commits into one?

**Pull request merged**